src/Algebra/Properties/CommutativeSemigroup.agda

```diff
@@ -132,3 +132,41 @@ xy∙z≈yz∙x x y z =  trans (xy∙z≈y∙zx x y z) (sym (assoc y z x))
 
 xy∙z≈zx∙y :  ∀ x y z → (x ∙ y) ∙ z ≈ (z ∙ x) ∙ y
 xy∙z≈zx∙y x y z =  trans (xy∙z≈z∙xy x y z) (sym (assoc z x y))
+
+------------------------------------------------------------------------------
+-- commutative semigroup has Jordan identity
+xy∙xx≈x∙yxx : ∀ x y → (x ∙ y) ∙ (x ∙ x) ≈ x ∙ (y ∙ (x ∙ x))
+xy∙xx≈x∙yxx x y = assoc x y ((x ∙ x))
+
+------------------------------------------------------------------------------
```

Comment:
I think we only need 1 section comment (i.e. `-----------------------`) for this group of identities?